frontend/src/metabase/visualizations/visualizations/SmartScalar/SettingsComponents/SmartScalarSettingsWidgets.styled.tsx

```diff
@@ -0,0 +1,17 @@
+import styled from "@emotion/styled";
+import { Button } from "metabase/ui";
+
+type ButtonProps = {
+  disabled: boolean;
+};
+
+export const ButtonStyled = styled(Button)<ButtonProps>`
+  padding-right: 0;
+  padding-left: 1rem;
+  width: 100%;
+
+  span {
+    width: 100%;
+    height: 100%;
+  }
+`;
```

Comment:
Have you tried Mantine's `getStylesRef` helper here?

Reply:
Just did. It works :tada: